Documentation/workflow/UsingYourBuild.md

```diff
@@ -201,22 +201,34 @@ windows (on Linux substitute ~/ for %HOMEPATH%) you could delete
 which should mke things work (but is fragile, confirm wile file timestamps that you are getting the version you expect)
 
 
-## Step 8.1 Quick updates in place.  
+## Step 8.1 (Optional) Quick updates in place.  
 
 The 'dotnet publish' step in step 6 above creates a directory that has all the files necessary to run your app
 including the CoreCLR and the parts of CoreFX that were needed.    You can use this fact to skip some steps if
 you wish to update the DLLs.   For example typically when you update CoreCLR you end up updating one of two DLLs
 
-* Coreclr.dll - Most modifications (with the exception of the JIT compiler and tools) that are C++ code update
+* coreclr.dll - Most modifications (with the exception of the JIT compiler and tools) that are C++ code update
   this DLL. 
 * System.Private.CoreLib.dll - If you modified C# it will end up here.  
 * System.Private.CoreLib.ni.dll - the native image (code) for System.Private.Corelib.   If you modify C# code
-you will want to update both of these together in the target installation.  
+you will want to update both of these together in the target installation.   
 
 Thus after making a change and building, you can simply copy the updated binary from the `bin\Product\<OS>.<arch>.<flavor>`
 directory to your publication directory (e.g. `helloWorld\bin\Debug\netcoreapp1.0\win7-x64\publish`) to quickly
-deploy your new bits.   
-
+deploy your new bits.   You can build just the .NET Library part of the build by doing (debug, for release add 'release qualifier)
+```bat
+    .\build skiptests skipnative 
```

Comment:
How does this compare to:
build windowsmscorlib release
?

I often also do:
build linuxmscorlib release
Etc.

You might also want to point out that these command lines are Windows specific, e.g. it would need to use ./build.sh ok Unix.